### PR TITLE
chore: update README.md

### PR DESCRIPTION
**Pull request by utkarsh006**
####  Description

- Updated README.md for better readability of users.
- Formatted lines of README.
- Shorten the README,  so that users can interact with it for longer periods of time.
- Added contributors section.
- Highlighted important text.
#### Screenshots

Screenshot (192)
Screenshot (188)
Screenshot (189)
Screenshot (190)
Screenshot (191)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,11 +3,13 @@
...
 =============================
 
-_The new Open Food Facts app is located at https://github.com/openfoodfacts/smooth-app_
+_The new Open Food Facts app is located [HERE](https://github.com/openfoodfacts/smooth-app)_
 
-**Note: this codebase is currently only deployed for Open Beauty Facts, Open Pet Food Facts and Open Products Facts apps.**
+**Note: This codebase is currently only deployed for Open Beauty Facts, Open Pet Food Facts and Open Products Facts apps.**
 
-Open Food Facts is collaborative food products database made by everyone, for everyone. Open Food Facts contributors gathers information and data on food products from around the world, using mobile apps.
+```Open Food Facts is collaborative food products database made by everyone, for everyone. Open Food Facts contributors gathers information and data on food products from around the world, using mobile apps.```
```
Comment (VaiTon) [inline README.md]: I would prefer to keep it as normal text

File: README.md [hunk 2]
```diff
@@ -131,3 +129,14 @@ The project was initially started by [Scot Scriven](https://github.com/itchix),
...
     See the License for the specific language governing permissions and 
     limitations under the License.
+</details>
+
+<br><br>
+
+## The project was started by [Scot Scriven](https://github.com/itchix) 😄
+
+## Contributors
+
```
Comment (VaiTon) [inline README.md]:
```suggestion
## Contributors

The project was started by [Scot Scriven](https://github.com/itchix) 😄

```

**Comment by utkarsh006** [comment]
@VaiTon I have made the changes. Kindly look into them.

**Comment by VaiTon** [comment]
Hey @utkarsh006, I left another comment

**Comment by utkarsh006** [comment]
> Hey @utkarsh006, I left another comment

Should I leave contributors section as it was earlier?

**Comment by utkarsh006** [comment]
@VaiTon I have done the changes told by you. Left the contributors section in its previous state. Do Review.

**Comment by VaiTon** [comment]
@utkarsh006 you can also add the contributors, I just commented about the original author being out of the contributors section

**Comment by utkarsh006** [comment]
> @utkarsh006 you can also add the contributors, I just commented about the original author being out of the contributors section

Finally done all the tasks. Hope, PR gets merged.

**Review by VaiTon — APPROVED** [review]

**Comment by utkarsh006** [comment]
@teolemon Kindly review my PR.

**Pull request merged**